src/datasets/precise/index.js

```diff
@@ -18,13 +18,30 @@ export default {
         },
         formation_energy: {
             label: "Formation energy",
+            filterAble: true,
```

Comment:
Change this to be `filterable`.

In the future, we can replace this with simply knowing that a variable is continuous, right?